transaction/transaction-utils.go

```diff
@@ -0,0 +1,72 @@
+package transaction
+
+import (
+	"github.com/wopta/goworkspace/models"
+	"time"
+)
+
+func ReinitializePaymentInfo(tr *models.Transaction) {
+	if tr.IsPay && !tr.IsDelete {
+		return
+	}
+	tr.IsPay = false
+	tr.IsDelete = false
+	tr.PaymentNote = ""
+	tr.PaymentMethod = ""
+	tr.PayDate = time.Time{}
+	tr.TransactionDate = time.Time{}
+	tr.Status = models.TransactionStatusToPay
+	tr.StatusHistory = append(tr.StatusHistory, "Reinitialized", models.TransactionStatusToPay)
```

Comment:
extract "Reinitialized" as an inner private constant in this file

const transactionStatusReinitialized string = "Reinitialized"